azure_jumpstart_ag/artifacts/AgLogonScript.ps1

```diff
@@ -151,6 +152,52 @@ else {
     Write-Host "ERROR: You have to fork the jumpstart-agora-apps repository!" -ForegroundColor Red
 }
 
+<# THIS CODE IS TEMPORARY COMMENTED DUE TO IMPORT ISSUES WITH SPN.
+#####################################################################
+# Import dashboard reports into Azure Data Explorer
+#####################################################################
+# Get Azure Data Explorer URI
+$adxEndPoint = (az kusto cluster show --name $adxClusterName --resource-group $resourceGroup --query "uri" -o tsv)
```

Comment:
regarding use of `az` instead of PowerShell cmdlets. 
This new code is the only place we're using `az` commands in the login script. 
I know that Dale and Lior were discussing a switch in some portion of the code to `az` due to problems with the cmdlets, but it would be good to sync on go-forward strategy to make sure we're consistent.